Extensions/CursorType.json

```diff
@@ -36,26 +36,125 @@
           "disabled": false,
           "folded": false,
           "type": "BuiltinCommonInstructions::JsCode",
-          "inlineCode": "var styleCursor = eventsFunctionContext.getArgument(\"style_cursor\") || \"default\" ;\nif (typeof document !== \"undefined\") {\n    document.body.style.cursor = styleCursor;\n}",
+          "inlineCode": "var styleCursor = eventsFunctionContext.getArgument(\"style_cursor\") || \"default\";\nif (typeof document !== \"undefined\") {\n    document.body.style.cursor = styleCursor;\n}",
           "parameterObjects": "",
-          "useStrict": true,
-          "eventsSheetExpanded": false
+          "useStrict": true
         }
       ],
       "parameters": [
         {
           "codeOnly": false,
-          "defaultValue": "",
-          "description": "The name of the new cursor (\"default\", \"pointer\", \"help\", etc...)",
-          "longDescription": "",
+          "defaultValue": "default",
+          "description": "The new cursor type",
+          "longDescription": "List of available cursors on https://developer.mozilla.org/en-US/docs/Web/CSS/cursor",
           "name": "style_cursor",
           "optional": false,
-          "supplementaryInformation": "",
-          "type": "string"
+          "supplementaryInformation": "[\"auto\", \"default\", \"none\", \"context-menu\", \"help\", \"pointer\", \"progress\", \"wait\", \"cell\", \"crosshair\", \"text\", \"vertical-text\", \"alias\", \"copy\", \"move\", \"no-drop\", \"not-allowed\", \"grab\", \"grabbing\", \"all-scroll\", \"col-resize\", \"row-resize\", \"n-resize\", \"e-resize\", \"s-resize\", \"w-resize\", \"ne-resize\", \"nw-resize\", \"se-resize\", \"sw-resize\", \"ew-resize\", \"ns-resize\", \"nesw-resize\", \"nwse-resize\", \"zoom-in\", \"zoom-out\"]",
+          "type": "stringWithSelector"
         }
       ],
       "objectGroups": []
     }
   ],
-  "eventsBasedBehaviors": []
+  "eventsBasedBehaviors": [
+    {
+      "description": "Sets the cursor appearence on hover of the object.",
```

Comment:
`appearance when the object is hovered`

Reply:
ping on this

Reply:
huh weird I could have sworn I had already fixed this

Reply:
No big deal, I merged and made a few fixes in another commit :)